modules/labs-react/common/lib/theming/useThemeRTL.ts

```diff
@@ -0,0 +1,23 @@
+import {ContentDirection, EmotionCanvasTheme, useTheme} from '@workday/canvas-kit-react/common';
```

Comment:
```suggestion
import {EmotionCanvasTheme, useIsRTL, useTheme} from '@workday/canvas-kit-react/common';
```
I'd like to reuse `useIsRTL` in `getThemedRTL` below.

Reply:
can do will need a small refactor to make hooks happy 👍